homeassistant/components/notify/__init__.py

```diff
@@ -46,12 +49,13 @@
 NOTIFY_SERVICE_SCHEMA = vol.Schema({
     vol.Required(ATTR_MESSAGE): cv.template,
     vol.Optional(ATTR_TITLE): cv.template,
+    vol.Optional(ATTR_MEDIA): cv.isfile,
```

Comment:
I attempted that approach, initially, but was not successful. I'll give it another try and look for examples.